dms/readme/DESCRIPTION.rst

```diff
@@ -2,3 +2,4 @@ DMS is a module for creating, managing and viewing document files directly
 within Odoo.
 This module is only the basis for an entire ecosystem of apps that extend and
 seamlessly integrate with the document management system.
+This module adds portal functionality for directories and files for allowed users, both portal or internal users. You can get as well a tokenized link from a directory or a file for sharing it with any anonymous user.
```

Comment:
For proper paragraph recognition in RST, you must leave an empty blank line in between, so please reformat this properly.